usr/local/www/services_dhcp_relay.php

```diff
@@ -40,13 +40,29 @@
 ##|-PRIV
 
 require("guiconfig.inc");
+require('classes/Form.class.php');
+
+function filterDestinationServers(array $destinationServers)
+{
+	return array_unique(
+		array_filter(
```

Comment:
I actually think if you're using array_filter with (!empty), you can remove the callback entirely, as the default of array_filter is remove everything that evals to false (which all empties do)